linux_os/guide/services/ntp/chronyd_specify_remote_server/ansible/shared.yml

```diff
@@ -0,0 +1,22 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+- (xccdf-var var_multiple_time_servers)
+
+- name: "Detect if chrony is already configured with pools or servers"
+  find:
+    path: /etc
+    patterns: chrony.conf
+    contains: '^[\s]*(?:server|pool)[\s]+[\w]+'
+  register: chrony_servers
+
+- name: "Add server configuration if none found in previous task"
```

Comment:
I don't feel confident about saying "previous task" in the task name. I'm not sure we need to incorporate the `when` clause in title.
Maybe just, `Configure remote time servers for chrony` ?